ios/OfflineManagerHolder.swift

```diff
@@ -0,0 +1,114 @@
+#if !os(tvOS)
```

Comment:
@rolandkakonyi How do you want to handle the fact that many of these classes are not available on tvOS?  I've started down the road of excluded it like this, similar to how it's done elsewhere in the project, but it seems a little cumbersome to do it this way.  If this is how we want to do it, I need to do it for the whole OfflineModule class, as well as the loadOfflineSource function in the PlayerModule.

Reply:
@joornby-angel Yes, unfortunately this is the only way we have now. These APIs are exclusively available on iOS only.